cmd/rookd/main.go

```diff
@@ -111,6 +113,8 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&cfg.dataDir, "data-dir", "/var/lib/rook", "directory for storing configuration")
 	rootCmd.PersistentFlags().StringVar(&logLevelRaw, "log-level", "INFO", "logging level for logging/tracing output (valid values: CRITICAL,ERROR,WARNING,NOTICE,INFO,DEBUG,TRACE)")
 	rootCmd.PersistentFlags().StringVar(&cfg.cephConfigOverride, "ceph-config-override", "", "optional path to a ceph config file that will be appended to the config files that rook generates")
+	rootCmd.PersistentFlags().StringVar(&cfg.containerVersion, "container-version", "latest", "version of the rook container to launch")
+	rootCmd.PersistentFlags().StringVar(&cfg.namespace, "namespace", "", "the namespace in which the operator is running")
```

Comment:
these two settings are needed by both the operator and api service, but you're right they should be on the specific commands. I'll update.